gobblin-modules/gobblin-http/src/main/java/gobblin/writer/AvroHttpWriterBuilder.java

```diff
@@ -34,8 +37,10 @@ public AvroHttpWriterBuilder fromConfig(Config config) {
     String urlTemplate = config.getString(HttpConstants.URL_TEMPLATE);
     String verb = config.getString(HttpConstants.VERB);
     String contentType = config.getString(HttpConstants.CONTENT_TYPE);
-    this.asyncRequestBuilder = new HttpRequestBuilder(urlTemplate, verb, contentType);
-    this.responseHandler = new ApacheHttpResponseHandler();
+    this.asyncRequestBuilder = new ApacheHttpRequestBuilder(urlTemplate, verb, contentType);
```

Comment:
Can we rename this class (AvroHttpWriterBuilder.java) to something with Apachexxx? Because we assume it's apache http client?